test/test_epm/test_gp.py

```diff
@@ -294,6 +294,12 @@ def test_predict_with_actual_values(self):
         # There's a slight difference between my local installation and travis
         self.assertLess(abs(var_hat[0][0] - 1121.8409184001594), 2)
 
+        # test other covariance results
+        _, var_hat = model.predict(X, cov_return_type='full_cov')
```

Comment:
Could you please check that the returned variances/stds for all three cases which are not none are numerically similar? I would like to make sure that we don't introduce stupid bugs here in the future.